x/mint/module/module.go

```diff
@@ -52,7 +51,6 @@ func (AppModuleBasic) RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
 // RegisterInterfaces registers the module's interface types
 func (b AppModuleBasic) RegisterInterfaces(r cdctypes.InterfaceRegistry) {
 	types.RegisterInterfaces(r)
-	v1beta1.RegisterInterfaces(r)
```

Comment:
should this be deleted? I thought we were keeping old msg servers and query servers around

Reply:
It had to be removed because the definition was deleted by proto gen.